benchmarks/run_offline.py

```diff
@@ -92,7 +92,8 @@ def main(argv):
 
   decode_state = engine.init_decode_state()
   profiler_started = False
-  for batch, _ in MAXTEXT_PREFILL.items():
+  for exp in range(4, 11):
+    batch = 2**exp
```

Comment:
The run_prefill_time function argument is seqlen, should we rename batch to seqlen?